geth/common/types.go

```diff
@@ -401,6 +402,8 @@ type NotifyResult struct {
 	Error  string `json:"error,omitempty"`
 }
 
+const passKey = "ACCOUNT_PASSWORD"
```

Comment:
Could you add a comment or rename this var into something so that it's clear it's an env var name?

Reply:
Ok. As you wish.